### ISSUE TITLE
Re-arrange Contact US header text

### ISSUE THREAD

**Issue by crarugal** [ukwa/ukwa-ui#141]
Contact Us page header should have additional text moved so that it sits under the header, to maintain a similar look to the other static pages.

Spawned from: Implement Recommended Background on Remaining Pages #120, 1.1.3 Alpha Iteration 3
https://github.com/ukwa/ukwa-ui/issues/120

https://dev.webarchive.org.uk/en/ukwa/contact

![image](https://user-images.githubusercontent.com/18530934/45364041-0f0b6f00-b5d1-11e8-9999-3f326ec2bbbb.png)

**Comment by nicolabingham** [comment]
Yes, this text ("To help answer any questions [......] please fill in the form below") should be moved to underneath the header to be consistent with the other static pages. Can it sit under the header before "Contact Form"?
